dll/win32/urlmon/CMakeLists.txt

```diff
@@ -4,7 +4,8 @@ add_definitions(
     -D_URLMON_
     -DENTRY_PREFIX=URLMON_
     -DPROXY_DELEGATION
-    -DWINE_REGISTER_DLL)
+    -DWINE_REGISTER_DLL
+    -DPROXY_CLSID_IS={0x79EAC9F1,0xBAF9,0x11CE,{0x8C,0x82,0x00,0xAA,0x00,0x4B,0xA9,0x0B}})
```

Comment:
OK, this actually matches what Wine does in their makefile: https://github.com/wine-mirror/wine/blob/5b3306a0d0aff221195ecc29872f25ed082eedd0/dlls/urlmon/Makefile.in#L31